src/TransactionalSession.MsSqlSystemDataClient.AcceptenceTests/When_using_transactional_session.cs

```diff
@@ -0,0 +1,228 @@
+﻿namespace NServiceBus.TransactionalSession.AcceptanceTests
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using System.Data.SqlClient;
+    using Microsoft.Extensions.DependencyInjection;
+    using NUnit.Framework;
+    using Persistence.Sql;
+
+    public class When_using_transactional_session : NServiceBusAcceptanceTest
+    {
+        [OneTimeSetUp]
+        public void OneTimeSetup()
+        {
+            MsSqlSystemDataClientConnectionBuilder.DropDbIfCollationIncorrect();
+            MsSqlSystemDataClientConnectionBuilder.CreateDbIfNotExists();
+        }
+
+        [TestCase(true)]
+        [TestCase(false)]
+        public async Task Should_send_messages_and_insert_rows_in_synchronized_session_on_transactional_session_commit(
+            bool outboxEnabled)
+        {
+            string rowId = Guid.NewGuid().ToString();
+
+            await Scenario.Define<Context>()
+                          .WithEndpoint<AnEndpoint>(s => s.When(async (_, ctx) =>
```

Comment:
even here the identation should be fixed and aligned with other acceptance tests

Reply:
I have done that already for Laila